docs/src/main/sphinx/installation/cli.rst

```diff
@@ -431,12 +431,13 @@ mode:
 .. _cli-output-format:
 
 Auto suggestion
```

Comment:
This should be moved outside of the `batch mode` section. Also, the anchor above refers to the `output formats` section. Looks like a result of a bad rebase.

Reply:
the anchor was messed up in the original documentation.. but yes.. we have to fix this

Reply:
You think it makes sense to move this to the 'Running the CLI' section?

Reply:
I'd move it after `history`, just before `batch mode`.

Reply:
that works